portal-ui/src/screens/Console/Users/UserDetails.tsx

```diff
@@ -49,16 +48,32 @@ import history from "../../../history";
 import UserServiceAccountsPanel from "./UserServiceAccountsPanel";
 import ChangeUserPasswordModal from "../Account/ChangeUserPasswordModal";
 import DeleteUserString from "./DeleteUserString";
-import ListItem from "@mui/material/ListItem";
-import ListItemText from "@mui/material/ListItemText";
-import List from "@mui/material/List";
 import LockIcon from "@mui/icons-material/Lock";
 import ScreenTitle from "../Common/ScreenTitle/ScreenTitle";
 import BoxIconButton from "../Common/BoxIconButton";
 import PanelTitle from "../Common/PanelTitle";
+import PageLayout from "../Common/Layout/PageLayout";
+import VerticalTabs from "../Common/VerticalTabs/VerticalTabs";
+import FormSwitchWrapper from "../Common/FormComponents/FormSwitchWrapper/FormSwitchWrapper";
+import BackLink from "../../../common/BackLink";
 
 const styles = (theme: Theme) =>
   createStyles({
+    pageContainer: {
+      border: "1px solid #EAEAEA",
+    },
+    tableHeight: {
+      height: "400px",
```

Comment:
```suggestion
      height: 400,
```

Please review table sizes, as I see that these values are hardcoded heights.